octomap_server/src/octomap_server.cpp

```diff
@@ -450,14 +447,8 @@ void OctomapServer::insertCloudCallback(const PointCloud2::ConstSharedPtr cloud)
           "You need to set the base_frame_id or disable filter_ground.");
     }
 
-
-    Eigen::Matrix4f sensor_to_base =
-      tf2::transformToEigen(sensor_to_base_transform_stamped.transform).matrix().cast<float>();
-    Eigen::Matrix4f base_to_world =
-      tf2::transformToEigen(base_to_world_transform_stamped.transform).matrix().cast<float>();
-
     // transform pointcloud from sensor frame to fixed robot frame
-    pcl::transformPointCloud(pc, pc, sensor_to_base);
+    pcl_ros::transformPointCloud(pc, pc, sensor_to_base_transform_stamped);
```

Comment:
@wxmerkt Do you have any reason to migrate transform method from pcl to pcl_ros?